src/main/src/main/java/org/geoserver/catalog/WMSLayerInfo.java

```diff
@@ -54,4 +54,12 @@
     public Set<StyleInfo> getStyles();
 
     public StyleInfo getDefaultStyle();
+
+    public Integer getMinScale();
```

Comment:
Scale denominators are normally  doubles in our code base (yes, one can specify a non integer one, maybe to try and match a tile matrix zoom level).

Reply:
will do